test/Microsoft.Health.Dicom.Tests.Integration/Persistence/SqlServerSchemaUpgradeTests.cs

```diff
@@ -40,6 +40,34 @@ public async Task GivenTwoSchemaInitializationMethods_WhenCreatingTwoDatabases_B
             await diffFixture.DisposeAsync();
         }
 
+        /// <summary>
+        /// There is small window where Sql schema is updated but not populated to web server, so the server still tries to call old stored procedure.
+        /// This test validate it works by checking stored procedure compatiblity. 
+        /// </summary>
+        /// <param name="schemaVersion">New schema version</param>
+        [Theory]
+        [MemberData(nameof(SchemaDiffVersions))]
+        public async Task GivenANewSchemaVersion_WhenApplying_ShouldBackCompatible(int schemaVersion)
+        {
+            int oldSchemaVersion = schemaVersion - 1;
+            // Create Sql store at old schema version
+            SqlDataStoreTestsFixture oldSqlStore = new SqlDataStoreTestsFixture(SqlDataStoreTestsFixture.GenerateDatabaseName($"COMPATIBLE_{oldSchemaVersion}_"), new SchemaInformation(oldSchemaVersion, oldSchemaVersion));
```

Comment:
nit: use `await using` since we're using `DisposeAsync`?

Reply:
want to leave a `evidence` for investigating if the test fail. :)